numpy/distutils/tests/test_system_info.py

```diff
@@ -0,0 +1,208 @@
+from __future__ import division, absolute_import, print_function
+
+import os
+from tempfile import mkstemp, mkdtemp
+
+import distutils
+from numpy.testing import *
```

Comment:
I'll try and hunt down what is needed, I merely copy-pasted from the other
test in the same directory (and then added the system_info import).

Will correct, thanks for the heads up, and sorry for the trouble!

2015-02-25 8:47 GMT+01:00 Ralf Gommers <notifications@github.com>:

> In numpy/distutils/tests/test_system_info.py
> <https://github.com/numpy/numpy/pull/5597#discussion_r25322878>:
>
> > @@ -0,0 +1,208 @@
> > +from __future__ import division, absolute_import, print_function
> > +
> > +import os
> > +from tempfile import mkstemp, mkdtemp
> > +
> > +import distutils
> > +from numpy.testing import *
>
> Can you import here only what's needed? Looks like TestCase and
> run_module_suite only now, but note that in general we prefer to use
> assert_ functions from numpy.testing rather than nose functions.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/numpy/numpy/pull/5597/files#r25322878>.
>



-- 
Kind regards Nick